spec/controllers/openid_connect/authorization_controller_spec.rb

```diff
@@ -0,0 +1,133 @@
+require 'rails_helper'
+
+RSpec.describe OpenidConnect::AuthorizationController do
+  let(:client_id) { 'urn:gov:gsa:openidconnect:test' }
+  let(:params) do
+    {
+      acr_values: 'http://idmanagement.gov/ns/assurance/loa/1',
+      client_id: client_id,
+      nonce:  SecureRandom.hex,
+      prompt: 'select_account',
+      redirect_uri: 'gov.gsa.openidconnect.test://result',
+      response_type: 'code',
+      scope: 'openid profile',
+      state:  SecureRandom.hex
+    }
+  end
+
+  describe '#index' do
+    subject(:action) { get :index, params }
+
+    context 'user is signed in' do
+      before do
+        user = create(:user, :signed_up)
+        sign_in user
```

Comment:
Is it possible to use `stub_sign_in` here? This would prevent a DB call, which will speed up the test. The difference in speed for this test alone is probably small, but when you consider all the controller tests we have, it all adds up.